### ISSUE TITLE
(sugestion 1.16.5) Add slot distribution of combs in multiblock centrifuges

### ISSUE THREAD

**Issue by NarigudoBR** [Team-Resourceful/ResourcefulBees#196]
Because why not? having the option to process 3x faster is a must have. Make it like the Mekanism factories, that you have the option to distribute or keep it in one slot

**Comment by Epic428** [comment]
Yes this has been asked for before. I would like to add it in. It would only be a temporary thing though since in 1.17 the centrifuge will be getting a second overhaul to make it far more dynamic than it is now.
